FWCore/ParameterSet/python/Config.py

```diff
@@ -533,11 +533,17 @@ def extend(self,other,items=()):
                 if not item.hasLabel_() :
                     item.setLabel(name)
             elif isinstance(item,Schedule):
-                self.__setattr__(name,item)
```

Comment:
Hi Chris,
I've assumed a use case like

```
f1 = cms.Process("Fragment")
f1.module1 = cms.EDProducer("Module", ...)
f1.path1 = cms.Path( f1.module1 )
f1.schedule = cms.Schedule( f1.Path )
```
```
f2 = cms.Process("Fragment")
f2.module2 = cms.EDProducer("Module", ...)
f2.path2 = cms.Path( f2.module2 )
f2.schedule = cms.Schedule( f2.Path )
```
```
process = cms.Process("Test")
process.extend(cms.ProcessWrapper(f1))
process.extend(cms.ProcessWrapper(f2))
```

The question is just, what do we want `process.schedule` to be.
Extending the original schedule feels more correct, but if you prefer to keep the existing behaviour and replace it, OK.
I will need a different workaround in cmsDriver.py, but it should not be a problem.